src/client/index.ts

```diff
@@ -20,25 +20,84 @@ export async function initClient(gameContainer: HTMLElement, client: Client) {
 			gameContainer.querySelector('button').onclick = () => {
 				client.call('start-matchmaking');
 			}
+
+
+			gameContainer.querySelectorAll('button')[1].onclick = () => {
```

Comment:
I would recommend adding classes 'play-vs-bot' and 'play-vs-human' to these buttons accordingly in main menu html and then use querySelector('.play-vs-bot') to get reference to that button. querySelectorAll you usually use when you want to iterate over all elements :)